data/.eslintrc.yaml

```diff
@@ -0,0 +1,19 @@
+extends: 'normandy'
```

Comment:
Sorry I wasn't clearer. This file will automatically build on top of the one in the parent directory. So you'll only have to include the changes for this directory. I imagine the whole file could be something like

```
globals:
  document: true
  self: true
```